WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInputViewModel.swift

```diff
@@ -99,7 +103,13 @@ final class ShippingLabelCustomsFormInputViewModel: ObservableObject {
         self.destinationCountry = destinationCountry
         self.allCountries = countries
         self.currency = currency
-        self.itemViewModels = customsForm.items.map { .init(item: $0, countries: countries, currency: currency) }
+        self.itemViewModels = customsForm.items.enumerated().map { index, element in
+                .init(item: element,
+                      countries: countries,
+                      currency: currency,
+                      shouldEnforceEUCustomsDescription: isEUShippingScenario,
+                      isEUTooltipAvailable: index == 0 && isEUShippingScenario)
```

Comment:
Should we also check the banner state, and only show the tooltip if the banner was dismissed? It felt a little weird to see it when I hadn't dismissed the banner, because I didn't know what it was signalling. It was especially weird to tap on it and not have anything happen:

https://github.com/woocommerce/woocommerce-ios/assets/8658164/99989880-b9c3-496b-9b0a-b502bacc5910

At the very least, if the tooltip is visible when the banner is also visible, we should probably disable it so it doesn't respond to taps.

Reply:
I believe the design idea for the tooltip to be displayed alongside the banner is to create a visual guide map of what the banner is talking about and the area that needs attention, so I think this situation is intentional. Still, I suggest we do this as a separate task because I don't see a straightforward way to share this banner visibility condition to the insides of the Custom item list UI without expanding the flag complexity introduced in this PR.

I am facing this: We signal to the Customs line list if the `EU Shipping Scenario` is active. If it's, every single line will adjust the description validation rule. Meanwhile, only the first line will receive the signal to show the tooltip. If I see the solution correctly, we must update the Custom line list every time the banner visibility changes or share the `@Published` flag containing the banner visibility state to the inside of the Customs line list. 

Is my assumption of possible solutions correct? Does it make sense to do it? Or maybe there's a simpler way I'm not detecting? 🤔

Reply:
One other possible solution I just thought here would be to scroll up and highlight the banner when the tooltip is clicked, even when the banner is already visible 🤔

Reply:
> I believe the design idea for the tooltip to be displayed alongside the banner is to create a visual guide map of what the banner is talking about and the area that needs attention, so I think this situation is intentional.

Ok, that's fair! In that case, it makes sense to keep the tooltip visible. That also avoids the complexity you mentioned — I can see how that would probably not be worth the effort to handle.

This is a really nice idea, though, maybe as a nice-to-have:

> One other possible solution I just thought here would be to scroll up and highlight the banner when the tooltip is clicked, even when the banner is already visible

That would resolve the confusion I had when I tapped the tooltip while the banner was still visible.

Reply:
Awesome! Thank you for sharing your thoughts, @rachelmcr! I created the iOS issue to track this enhancement [here](https://github.com/woocommerce/woocommerce-ios/issues/9776), and also created one for the [Android side](https://github.com/woocommerce/woocommerce-android/issues/9077) too.